calcom/cal.com
# feat: show ooo forwarding and emoji when not available

## What does this PR do?



- Fixes #17601 (GitHub issue number)
- Fixes CAL-4704 (Linear issue number - should be visible at the bottom of the GitHub issue description)




https://github.com/user-attachments/assets/b4903a6d-2b91-4994-b7dd-8d8966abbd27


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Create a busy event on your calendar for a specific date.
- Mark yourself as "Out of Office" (OOO) for the same date as the busy event.
- Navigate to the booking page for an event.
- Verify that the date is now marked as OOO, even though it is unavailable due to the busy event on the calendar.
